Formula/openiked.rb

```diff
@@ -0,0 +1,54 @@
+class Openiked < Formula
+  desc "IKEv2 daemon - portable version of OpenBSD iked"
+  homepage "https://openiked.org"
+  url "https://cdn.openbsd.org/pub/OpenBSD/OpenIKED/openiked-7.2.tar.gz"
+  mirror "https://mirror.edgecast.com/pub/OpenBSD/OpenIKED/openiked-7.2.tar.gz"
+  sha256 "55dc270bc40a121f855d949a25a5ffaeb11e7607e8198ec52160ef54b6946845"
+  license "ISC"
+
+  depends_on "cmake" => :build
+  depends_on "libevent"
+  depends_on "openssl@1.1"  # openssl@3 conflicts with libevent
+
+  on_linux do
+    depends_on "bison"
+  end
+
+  def install
+    system "cmake", "-DHOMEBREW=true",
+                    "-DCMAKE_BUILD_TYPE=Release",
+                    "-DCMAKE_INSTALL_SYSCONFDIR=#{etc}",
+                    "-DCMAKE_INSTALL_MANDIR=#{man}"
```

Comment:
I think this will work too
```suggestion
    system "cmake", "-DHOMEBREW=true", *std_cmake_args
```

Reply:
as I can understand from here https://github.com/Homebrew/brew/blob/master/Library/Homebrew/formula.rb#L1539 , SYSCONFDIR and MANDIR are not set with *std_cmake_args. For the CMake in OpenIKED to work correctly with the directory structure of homebrew, I think we need those flags.

Reply:
Aren't those normally based on the `CMAKE_INSTALL_PREFIX` ?

Reply:
Yes, they are. but the man-files won't be installed to `${prefix}/share/man` but in `${prefix}/man`, which is wrong according to https://docs.brew.sh/Formula-Cookbook#manuals
Also the whole config including keys would be relative to the package-path (`homebrew/cellar/openiked/7.2/etc/iked/`) instead of global inside homebrew (`homebrew/etc/iked/`). I think the second option makes more sense, because you don't want your keys and config relative to the package version.